projects/cli-files/go-cat/cmd/root.go

```diff
@@ -0,0 +1,18 @@
+package cmd
+
+import (
+	"log"
+	"os"
+)
+
+func Execute() {
+	args := os.Args[1:]
+	for _, v := range args {
+		data, err := os.ReadFile(v)
```

Comment:
This will read the whole file into memory, which may be a problem e.g. if your computer has 8GB RAM and you're trying to read a 100GB video file. Can you find a way to achieve this same effect _without_ having to read the whole file into memory at once?

Reply:
I didn't give much thought to this initially. Using scanner achieves this same effect but reads data in tokens.